customdisplayprofiles

```diff
@@ -1,84 +1,42 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
I think here we'd want to do `#!/usr/bin/env python3` so that it is possible to work in the context of a virtualenv which has `pyobjc` installed. In your testing with this, do you recall if you'd had pyobjc installed globally to Apple's python3 site-packages?

Reply:
Makes sense to me, as I did have to install pyobjc globally. Change made. (Sorry about the double commit)